tests/mock/clitools.rs

```diff
@@ -207,6 +207,16 @@ pub fn expect_not_stdout_ok(config: &Config, args: &[&str], expected: &str) {
     }
 }
 
+pub fn expect_not_stderr_ok(config: &Config, args: &[&str], expected: &str) {
```

Comment:
I'd prefer if this function were in a separate commit with a message explaining what the function is and why it's there.

i.e. something like:

```
clitools.rs: Add expect_not_stderr_ok

In order to support testing commands which may return certain
messages on `stderr` in a way in which they should *NOT*
produce such messages, add a counterpart to `expect_not_stdout_ok`.
```